apps/test-low-memory.sh

```diff
@@ -59,4 +59,6 @@ npm run lint
   ${CODECOV} -cF integration) ${LOG} log/appLabgameLabTest.log
 (PORT=9882 LEVEL_TYPE='craft' $GRUNT_CMD karma:integration && \
   ${CODECOV} -cF integration) ${LOG} log/craftTest.log
+(PORT=9883 LEVEL_TYPE='spritelab' $GRUNT_CMD karma:integration && \
+    ${CODECOV} -cF integration) ${LOG} log/spriteLabTest.log    
```

Comment:
Thank you!! For history: I messed up by not adding this line in PR https://github.com/code-dot-org/code-dot-org/pull/23702, so this test _never_ ran as part of the CI build.

`npm run test:integration` would run this test, but not `test-low-memory.sh` (until now).